contracts/test/FleekERC721.ts

```diff
@@ -1,935 +0,0 @@
-import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
```

Comment:
Do you guys know why this file is being shown as deleted? Look, in `develop` right now there is no file like this https://github.com/fleekxyz/non-fungible-apps/tree/develop/contracts/test but for some reason this PR is deleting it.  I just want to make sure it's not coming from some commit based off another branch than develop for example.

Reply:
This branch is based on `develop`; only hardhat tests and foundry tests refactor branches were merged. It might have come from them.